src/Compilers/CSharp/Portable/BoundTree/BoundNodes.xml

```diff
@@ -435,6 +435,12 @@
     <Field Name="GetTypeFromHandle" Type="MethodSymbol" Null="allow"/>
   </Node>
 
+  <Node Name="BoundMethodToken" Base="BoundExpression">
```

Comment:
This looks very similar to `BoundMethodToken` below.  We're not sharing the representation because we need to be able to distinguish this use case?  Would it be possible to use the presence or absence of `GetMethodFromHandle` as the distinguishing factor?